Makefile

```diff
@@ -213,3 +116,6 @@ docker_test:
 
 docker_unit_test:
 	go run test.go -flavor $(flavor) unit
+
+reshard_tests:
```

Comment:
Can you please add a comment here?

Otherwise, some people may confuse that these will call tests which test the Vitess resharding functionality :)